requirements-dev.txt

```diff
@@ -7,5 +7,6 @@ behave
 pexpect
 coverage==4.3.4
 codecov==2.0.9
-git+https://github.com/hayd/pep8radius.git@c8aebd0e1d272160896124e104773b97a6249c3e#egg=pep8radius
+autopep8==1.3.3
```

Comment:
I couldn't get the linter to run with autopep8 1.3.4, which was recently released.

For example: https://travis-ci.org/dbcli/mycli/jobs/337987355